about.html

```diff
@@ -86,66 +65,15 @@
-							<a href="http://eepurl.com/bonKaL"><i class='black large bordered mail icon'></i></a>
-							<a href="http://github.com/NebulousLabs/Sia"><i class='black large bordered github alternate icon'></i></a>
-							<br> We also invite you to #siatalk on freenode.
-						</div>
-					</div>
-					<div class='col-md-6'>
-						<div class='download-links'>
-							Download Sia for
-							<a href="https://github.com/NebulousLabs/Sia-UI/releases/download/v0.3.3.1-beta/Sia-UI-v0.3.3.1-beta-win64.zip" class='link'>Windows 64bit</a>
-							<span style="font-size: 0.8em">&nbsp;(<a href="https://github.com/NebulousLabs/Sia/releases/download/v0.3.3.1-beta/Sia-v0.3.3.1-beta-windows-amd64.zip" class='link'>CLI</a>)</span>
-							<br>
-							Download Sia for
-							<a href="https://github.com/NebulousLabs/Sia-UI/releases/download/v0.3.3.1-beta/Sia-UI-v0.3.3.1-beta-linux64.tar.xz" class='link'>Linux 64bit</a>
-							<span style="font-size: 0.8em">&nbsp;(<a href="https://github.com/NebulousLabs/Sia/releases/download/v0.3.3.1-beta/Sia-v0.3.3.1-beta-linux-amd64.tar.gz" class='link'>CLI</a>)</span>
-							<br>
-							Download Sia for
-							<a href="https://github.com/NebulousLabs/Sia-UI/releases/download/v0.3.3.1-beta/Sia-UI-v0.3.3.1-beta-darwin.tar.xz" class='link'>OS X 64bit</a>
-							<span style="font-size: 0.8em">&nbsp;(<a href="https://github.com/NebulousLabs/Sia/releases/download/v0.3.3.1-beta/Sia-v0.3.3.1-beta-darwin-amd64.zip" class='link'>CLI</a>)</span>
-							<br>
-							Download Sia for
-							<a href="https://github.com/NebulousLabs/Sia-UI/releases/v0.3.3.1-beta/" class='link'>Other Platforms</a>
-							<span style="font-size: 0.8em">&nbsp;(<a href="https://github.com/NebulousLabs/Sia/releases/v0.3.3/" class='link'>CLI</a>)</span>
-						</div>
-					</div>
-				</div>
-			</div>
-		</div>
-		<div class='row'>
-			<div class='col-xs-12'>
-				<div class='footer'>
-					<div class='copyright'>&copy; Nebulous Inc.</div>
-					<div class='footer-links'>
-						<span class='link'><a href="index.html">Home</a></span>
-						<span class='link'><a href="about.html">About</a></span>
-						<span class='link'><a href="jobs.html">Jobs</a></span>
-						<span class='link'><a href="sia.pdf">Whitepaper</a></span>
-						<span class='link'><a href="http://github.com/NebulousLabs/Sia">GitHub</a></span>
-					</div>
-				</div>
-			</div>
+
+			<!--#include file="/ssi/pagefooter.html" -->
+
 		</div>
-	<!-- Google Analytics -->
-	<script>
-		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-		ga('create', 'UA-58715089-1', 'auto');
-		ga('send', 'pageview');
-	</script>
+
+		<!--#include file="/ssi/backgroundfooter.html" -->
+
+		<!--#include file="/ssi/googleanalytics.html" -->
```

Comment:
also, is it okay to smash these together?

```
<!--#include ... -->
<!--#include ... -->
```

 As opposed to having a break between them.